src/Microsoft.Data.Analysis/DateTimeComputation.cs

```diff
@@ -311,4 +311,295 @@ public void Round(PrimitiveColumnContainer<DateTime> column)
         }
 
     }
+
+    internal class DateTimeArithmetic : IPrimitiveDataFrameColumnArithmetic<DateTime>
```

Comment:
It looks like all other primitive-type Arithmethic class are generated from .tt, is there any reason we don't follow that pattern for `Datetime`

Reply:
Yes you're right. I should move this to the generated Arithmetic class. I'll need to double check on the Computation class that is the other half of this file